setup.cfg

```diff
@@ -23,7 +23,7 @@ classifiers =
 package_dir =
     = epispot
 packages = find:
-python_requires = >=3.6
+python_requires = >=3.7
```

Comment:
Why deprecate the support for 3.6?

Reply:
Python 3.6 is no longer included in our testing suite as a result of #93. Additionally, this change was already made to `setup-nightly.py` a long time ago. This is really a minor change and has already been detailed in the CHANGELOG.